index.js

```diff
@@ -91,6 +91,7 @@ const plugins = [
       { yes: "GitHub" },
       { no: "hostname", yes: "host name" },
       { yes: "JavaScript" },
+      { yes: "TypeScript" },
```

Comment:
The current list is in alphabetical order. Can we please retain that by moving this just above "Unix"?

Other than that, looks good to me.